src/native/libs/configure.cmake

```diff
@@ -458,6 +458,18 @@ check_symbol_exists(
     sys/epoll.h
     HAVE_EPOLL)
 
+check_symbol_exists(
+    gethostname
+    unistd.h
+    HAVE_GETHOSTNAME)
+
+check_symbol_exists(
+    getnameinfo
+    netdb.h
+    HAVE_GETNAMEINFO)
+
+check_struct_has_member("struct sockaddr_un" sun_path "syn/un.h" HAVE_SOCKADDR_UN_SUN_PATH)
```

Comment:
What's wrong with this ? It's false on OSX and maybe elsewhere. That breaks `SystemNative_GetDomainSocketSizes`